ngraph/core/include/ngraph/op/multiclass_nms.hpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/util/nms_base.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v8
+        {
+            /// \brief MulticlassNms operation
+            ///
+            class NGRAPH_API MulticlassNms : public util::NmsBase
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+
+                MulticlassNms() = default;
+
+                /// \brief Constructs a MulticlassNms operation
+                ///
+                /// \param boxes Node producing the box coordinates
+                /// \param scores Node producing the box scores
+                /// \param sort_result Specifies order of output elements
+                /// \param sort_result_across_batch Specifies  whenever it is necessary to
+                /// sort selected boxes across batches or not
+                /// \param output_type Specifies the output tensor type
+                /// \param iou_threshold Specifies intersection over union threshold
+                /// \param score_threshold Specifies minimum score to consider box for the
+                /// processing
+                /// \param nms_top_k Specifies maximum number of boxes to be selected per
+                /// class, -1 meaning to keep all boxes
+                /// \param keep_top_k Specifies maximum number of boxes to be selected per
+                /// batch element, -1 meaning to keep all boxes
+                /// \param background_class Specifies the background class id, -1 meaning to keep
+                /// all classes
+                /// \param nms_eta Specifies eta parameter for adpative NMS, in close range [0, 1.0]
+                /// \param normalized Specifies whether boxes are normalized or not
+                MulticlassNms(const Output<Node>& boxes,
+                              const Output<Node>& scores,
+                              const SortResultType sort_result_type = SortResultType::NONE,
+                              bool sort_result_across_batch = false,
+                              const ngraph::element::Type& output_type = ngraph::element::i64,
+                              const float iou_threshold = 0.0f,
+                              const float score_threshold = 0.0f,
+                              const int nms_top_k = -1,
+                              const int keep_top_k = -1,
+                              const int background_class = -1,
+                              const float nms_eta = 1.0f,
+                              const bool normalized = true);
```

Comment:
Could you create a structure with all attributes of the operation to decrease number of arguments of the constructor?
```suggestion
            class NGRAPH_API MulticlassNms : public util::NmsBase
            {
            public:
                NGRAPH_RTTI_DECLARATION;
                
                struct Attributes
                {
                              SortResultType sort_result_type = SortResultType::NONE,
                              sort_result_across_batch = false,
                              ngraph::element::Type& output_type = ngraph::element::i64,
                              float iou_threshold = 0.0f,
                              float score_threshold = 0.0f,
                              int nms_top_k = -1,
                              int keep_top_k = -1,
                              int background_class = -1,
                              float nms_eta = 1.0f,
                              bool normalized = true
                };

                MulticlassNms() = default;                            

                /// \brief Constructs a MulticlassNms operation
                ///
                /// \param boxes Node producing the box coordinates
                /// \param scores Node producing the box scores
                /// \param attrs Attributes of the operation
                MulticlassNms(const Output<Node>& boxes,
                              const Output<Node>& scores,
                              const Attributes& attrs);

                bool visit_attributes(AttributeVisitor& visitor) override;

                std::shared_ptr<Node>
                    clone_with_new_inputs(const OutputVector& new_args) const override;

                Attributes get_attrs() const { return m_attrs; }
                void set_attrs(const Attributes& attrs)
                {
                    m_attrs = attrs;
                }

            protected:
                Attributes m_attrs;
                
                void validate() override;
            };
```
The cpp-file should be correspondingly changed. Of course, we need comments before structure `Attributes`.

Reply:
Done